files/architecture/backend_architecture.tex

```diff
@@ -0,0 +1,15 @@
+\section{Back-end architecture}
+The back-end application is written in TypeScript and build with a framework called Apollo.
+The back-end is spun up using Express and \gls{node}
+
+The back-end contains 3 layers of abstraction.
+
+\graphic{1}{backend_architecture}{Back-end layer}{fig:backendLayer}
```

Comment:
Maybe make the image description more explanatory. Something like: "The back-end architecture layers"